scripts/generate_header.py

```diff
@@ -409,7 +410,10 @@ def main() -> int:
     for index, dev_om in devices_om:
         reglist = find_registers(dev_om)
         intlist = find_interrupts(dev_om, device)
-        base_int = min(i.number for i in intlist)
+        if len(intlist) == 0:
+            base_int = None
+        else:
+            base_int = min(i.number for i in intlist)
```

Comment:
This can be replaced with:

```python
base_int = min(i.number for i in intlint, default=None)
```

The `default` keyword argument allows you to specify a value if the iterable is empty. See https://docs.python.org/3/library/functions.html#min

Could you write a short description of how this was tested? Just want to make sure that this covers everything that needs to be changed if there are no interrupts.

Reply:
fixed, added explicit test to show how I tested it.